REFERENCES.md

```diff
@@ -0,0 +1,184 @@
+	Year = {2019}
+}
+```
+A flexible way to model probabilistic forecasts via spline quantile forecasts.
+```
+@inproceedings{gasthaus2019probabilistic,
+	Author = {Gasthaus, Jan and Benidis, Konstantinos and Wang, Yuyang and Rangapuram, Syama Sundar and Salinas, David and Flunkert, Valentin and Januschowski, Tim},
+	Booktitle = {The 22nd International Conference on Artificial Intelligence and Statistics},
+	Pages = {1901--1910},
+	Title = {Probabilistic Forecasting with Spline Quantile Function RNNs},
+	Year = {2019}
+}
+```
+Using RNNs to parametrize State Space Models.
+```
+@inproceedings{rangapuram2018deep,
+	Author = {Rangapuram, Syama Sundar and Seeger, Matthias W and Gasthaus, Jan and Stella, Lorenzo and Wang, Yuyang and Januschowski, Tim},
+	Booktitle = {Advances in Neural Information Processing Systems},
+	Pages = {7785--7794},
+	Title = {Deep state space models for time series forecasting},
+	Year = {2018}
+}
+```
+A scalable state space model. Note that code for this model
+is currently not available in GluonTS.
+```
+@inproceedings{seeger2016bayesian,
+	Author = {Seeger, Matthias W and Salinas, David and Flunkert, Valentin},
+	Booktitle = {Advances in Neural Information Processing Systems},
+	Date-Added = {2019-06-27 13:17:25 +0000},
+	Date-Modified = {2019-06-27 13:18:01 +0000},
+	Pages = {4646--4654},
+	Title = {Bayesian intermittent demand forecasting for large inventories},
+	Year = {2016}
+}
+```
+
+
+
+## Tutorials
+Tutorials are available in bibtex and with accompanying material,
+ in particular slides, linked from below.
+[Tutorial at KDD 2019](https://lovvge.github.io/Forecasting-Tutorial-KDD-2019/)
+```
+@inproceedings{faloutsos19forecasting2,
+  author    = {Faloutsos, Christos and
+               Flunkert, Valentin and
+               Gasthaus, Jan and
+               Januschowski, Tim and
+               Wang, Yuyang},
+  title     = {Forecasting Big Time Series: Theory and Practice},
+  booktitle = {Proceedings of the 25th {ACM} {SIGKDD} International Conference on
+               Knowledge Discovery {\&} Data Mining, {KDD} 2019, Anchorage, AK,
+               USA, August 4-8, 2019.},
+  year      = {2019}
+  }
+```
+[Tutorial at SIGMOD 2019](https://lovvge.github.io/Forecasting-Tutorials/SIGMOD-2019/)
+```
+@inproceedings{faloutsos2019forecasting,
+ author = {Faloutsos, Christos and Gasthaus, Jan and Januschowski, Tim and Wang, Yuyang},
+ title = {Classical and Contemporary Approaches to Big Time Series Forecasting},
+ booktitle = {Proceedings of the 2019 International Conference on Management of Data},
+ series = {SIGMOD '19},
+ publisher = {ACM},
+ address = {New York, NY, USA},
+ year = {2019}
+} 
+```
+[Tutorial at VLDB 2018](https://lovvge.github.io/Forecasting-Tutorial-VLDB-2018/)
+```
+@article{faloutsos2018forecasting,
+	Author = {Faloutsos, Christos and Gasthaus, Jan and Januschowski, Tim and Wang, Yuyang},
+	Date-Added = {2019-07-24 13:47:16 +0000},
```

Comment:
These fields are not required (they appear in other entries too)

Reply:
yes.